packages/lesswrong/components/sequences/BooksProgressBar.tsx

```diff
@@ -84,12 +84,12 @@ const BooksProgressBar = ({ book, classes }: {
     <div className={classes.bookProgress}>
       {
         bookPosts.map(post => (
-          <LWTooltip key={post._id} title={<PostsPreviewTooltip post={post}/>} tooltip={false} flip={false}>
+          <PostsTooltip post={post} key={post._id}>
```

Comment:
It was there before, but I removed it because all it does (as far as I can tell) is create a bug where the tooltip goes off the edge of the screen and can't be read.